arrows/ffmpeg/tests/test_video_input_ffmpeg.cxx

```diff
@@ -30,273 +30,274 @@ static int TOTAL_NUMBER_OF_FRAMES = 50;
 
 // ----------------------------------------------------------------------------
 int
-main(int argc, char* argv[])
+main( int argc, char* argv[] )
 {
   ::testing::InitGoogleTest( &argc, argv );
   TEST_LOAD_PLUGINS();
 
-  GET_ARG(1, g_data_dir);
+  GET_ARG( 1, g_data_dir );
 
   return RUN_ALL_TESTS();
 }
 
 // ----------------------------------------------------------------------------
 class ffmpeg_video_input : public ::testing::Test
 {
-  TEST_ARG(data_dir);
+  TEST_ARG( data_dir );
 };
 
 // ----------------------------------------------------------------------------
-TEST_F(ffmpeg_video_input, create)
+TEST_F ( ffmpeg_video_input, create )
```

Comment:
Not necessarily; these are rather odd since it doesn't quite match C++ syntax and needs `TEST`, `TEST_F`, etc. to be manually assigned a token type that will format correctly. It seems that process is breaking down, possibly because `FUNC_WRAP` is not a suitable token type. Possibly we should use `FUNC_PROTO` instead? (Note: this is stuff in the config; in particular, see the bottom of the config.)